the-net-pizza-html/piatti-speciali/michela-loddo/index.html

```diff
@@ -0,0 +1,156 @@
+    <ul>
+        <li><a href=https://it.wikipedia.org/wiki/Asti_(vino)>Moscato D'asti</a>: questo vino dolce e frizzante con aromi di pesca, 
+            albicocca e agrumi complementerebbe bene la dolcezza della salsa d'arancio e il sapore fruttato dei mirtilli.</li>
+        <li><a href=https://it.wikipedia.org/wiki/Brachetto_d%27Acqui>Brachetto d'Acqui</a>: un altro vino dolce e frizzante con 
+            sentori di fragole e lamponi. La sua dolcezza equilibrata e il carattere fruttato si sposano bene con il dessert.</li>
+        <li><a href=https://it.wikipedia.org/wiki/Riesling>Riesling dolce</a>: un Riesling dolce con note di mela, pesca e agrumi
+            può offrire un bel contrasto con l'acidità dei mirtilli e sottolineare i sapori dell'arancia.</li>
+        <li><a href=https://it.wikipedia.org/wiki/Vin_santo_toscano>Vin Santo</a>: questo vino italiano è dolce e ricco, con sentori
+            di noci, uvetta e miele. Il suo profilo aromatico complesso si abbina bene al gusto intenso della salsa d'arancio e dei
+            mirtilli.</li>
+        <li><a href=https://it.wikipedia.org/wiki/Prosecco_(vino)>Spumante demi-sec o prosecco dolce</a>: un vino frizzante leggermente
+            dolce può essere una scelta fresca e vivace per accompagnare le crêpe. Contribuirà a pulire il palato e ad accentuare i
+            sapori dei frutti.</li>
+        </ul>
+
+    <h2>Tabella nutrizionale</h2>
+    <table border="1"> <!-- Errore validatore HTML: dovremmo usare CSS -->
+        <thead>
+            <tr>
+                <th>Analisi nutrizionale media</th>
+                <th>Unità</th>
+                <th>Per 100 gr</th>
+                <th>Per pezzo</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Valore energetico</td>
+                <td>Kcal</td>
+                <td>371</td>
+                <td>100</td>
+            </tr>
+            <tr>
+                <td>Grassi</td>
+                <td>grammi</td>
+                <td>4,7</td>
+                <td>1,3</td>
+            </tr>
+            <tr>
+                <td>Carboidrati</td>
+                <td>grammi</td>
+                <td>15</td>
+                <td>4,1</td>
+            </tr>
+            <tr>
+                <td>Fibre</td>
+                <td>grammi</td>
+                <td>&lt;0,5</td>
+                <td>&lt;0,5</td>
+            </tr>
+            <tr>
+                <td>Proteine</td>
+                <td>grammi</td>
+                <td>65</td>
+                <td>17</td>
+            </tr>
+            <tr>
+                <td>Sale</td>
+                <td>grammi</td>
+                <td>1,5</td>
+                <td>0,40</td>
+            </tr>
+
+        </tbody>
+    </table>
+
+    <form>
+        <input type="button"
+         value="Back to the main page"
+         onClick="window.location.href='http://127.0.0.1:5500/the-net-pizza-html/index.html';"
```

Comment:
Ti conviene usare sempre url relativi, quindi in questo caso `/the-net-pizza-html/index.html`